### ISSUE TITLE
IoTedge Security Daemon Uninstall operation does not complete

### ISSUE THREAD

**Issue by alkreddy** [Azure/iotedge#793]
## Expected Behavior
Uninstall-SecurityDaemon should complete with proper cleanup of resources

## Current Behavior
Uninstall-SecurityDaemon is blocked on step 'Deleting Moby data root directory....'. Waited for 45 minutes

## Steps to Reproduce

1. Install IoTEdge Security Daemon for windows containers
2. Work through some IoT Module development spanning building, pushing and pulling images from registry. Run the modules
3. Stop Security Daemon
4. Run Uninstall script

### Device (Host) Operating System

Windows 10 Pro
### Architecture

amd64
### Container Operating System

Windows Containers
#### iotedged

1.0.6
#### Edge Agent

1.0.6
## Logs

```
PS C:\Users\user> . {Invoke-WebRequest -useb aka.ms/iotedge-win} | Invoke-Expression; `
>> Uninstall-SecurityDaemon -DeleteConfig -DeleteMobyDataRoot

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     0.0        IotEdgeSecurityDaemon               {Install-SecurityDaemon, Uninstall-SecurityDaemon}
Uninstalling...
Deleting install directory 'C:\ProgramData\iotedge'...
Deleting Moby data root directory 'C:\ProgramData\iotedge-moby-data'...
```
The script waits here indefinitely. 
Tried break and repeat. No change.

**Comment by alkreddy** [comment]
Left the script running for over 10 hrs. It doesn't timeout / complete / delete files in Moby data root

**Comment by alkreddy** [comment]
There is a panic.log file in moby root and it has the following entries....
```
http: multiple response.WriteHeader calls
http: multiple response.WriteHeader calls
```

**Comment by alkreddy** [comment]
@veyalla , Sry of the delayed response. Nope. Reboot did not help either. To complete the uninstall I ran the script without `-DeleteMobyRoot` option

**Comment by jenoola** [comment]
@veyalla I'm running into the same issue, and my iotedge configuration is the same

**Comment by veyalla** [comment]
@jenoola This behavior seems specific to some systems. The only workaround we found is to uninstall without the ```-DeleteMobyRoot``` switch. We're making Windows packaging improvements to address this, it should land in the next release or two.

**Comment by alkreddy** [comment]
@veyalla  This issue appears to occur specifically after using the system to build edge modules, publish and push them to hub, pull and run the modules. On the same computer, when I first installed the security daemon and later uninstalled without any further usage of the edge components, the uninstall was smooth and clean.

**Comment by veyalla** [comment]
@alkreddy Thanks for this information, I'll make a note to test this scenario with our planned packaging improvements.

**Comment by arsing** [comment]
It might've been blocked at the `takeown` step of deleting the Moby data root, in which case it will be fixed by #910. If you get a repro, try pressing <kbd>Y</kbd> and see if the script continues.

**Comment by myagley** [comment]
This has been released as part of the 1.0.7 release candidate. The 1.0.7 release contains an all new windows installer based on cab files that is much more resilient. You can get packages here: https://github.com/Azure/azure-iotedge/releases/tag/1.0.7-rc2
